services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -163,6 +163,25 @@ module.exports = {
     },
   },
 
+
+  /**
+   *
+   */
+  ContentVenue: {
+    spaces: ({ _id: venue }, { input }, { basedb }, info) => {
```

Comment:
This should use the `@refMany` directive with different `localField` and `foreignField` values